clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp

```diff
@@ -932,18 +932,18 @@ static Expected<StringRef> runLLVMToSPIRVTranslation(StringRef File,
 }
 
 /// Adds all AOT backend options required for SYCL AOT compilation step to
-/// 'CmdArgs'.
-/// 'Args' encompasses all arguments required for linking and wrapping device
-/// code and will be parsed to generate backend options required to be passed
-/// into the SYCL AOT compilation step.
-/// IsCPU is a bool used to direct option generation. If IsCPU is false, then
-/// options are generated for AOT compilation targeting Intel GPUs.
-static void addBackendOptions(const ArgList &Args,
-                              SmallVector<StringRef, 8> &CmdArgs, bool IsCPU) {
-  StringRef OptC =
-      Args.getLastArgValue(OPT_sycl_backend_compile_options_from_image_EQ);
+/// \p CmdArgs.
+/// \p Args encompasses all arguments required for linking and wrapping device
+/// code.
+/// \p IsCPU is a bool used to distinguish whether the target is an Intel GPU or
+/// Intel CPU.
+/// \p BackendOptions is a string containing options that are going to be parsed
```

Comment:
The LLVM coding standards emphasize clear, concise, and `informative comments` with the goal to explain `what` the code is doing and `why`, rather than simply restating the obvious **how**.

This comment : `BackendOptions is a string containing options that are going to be parsed and inserted in \p CmdArgs`. is simply stating how it is handled in the code. It would help to describe what are `BackendOptions` and why are they being passed here as an argument to this function.

Please make sure the `what` and `why` aspects of the commenting guidelines are followed throughout this patch.

Reply:
I adjusted them.